README.md

```diff
@@ -1 +1,52 @@
-# hello-microverse
\ No newline at end of file
+
+# hello-microverse
+
+> Description the project.
+
+
+## Built With
+
+- HTML & CSS
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
```

Comment:
- [ ] **Required** lets add the code to clone this repo. For example:- 
` git clone https://github.com/Achilles-Dev/hello-microverse.git`